src/components/settings/ReverseProxy.vue

```diff
@@ -20,11 +20,16 @@
             </div>
 
             <div v-if="errorMessage" class="mt-3">
-                Message:
+                {{ $t("Error Message:") }}
```

Comment:
Since cloudflared write all output to `stderr` (standard error), it is acutally not an error message. `Message` would be better.